v2/emailpassword/custom-ui/handling-session-tokens.mdx

```diff
@@ -209,6 +210,95 @@ fileprivate class NetworkManager {
 By default our mobile SDKs use a bearer token in the Authorization header to provide credentials.
 :::
 
+### Getting the access token
```

Comment:
Add to session recipe:
- With ui switcher and copy section

Add to session section in all other recipes (Same page as above, using copy docs)